packages/transformers/babel/package.json

```diff
@@ -24,6 +24,7 @@
     "@babel/plugin-transform-typescript": "^7.4.5",
     "@babel/preset-env": "^7.0.0",
     "@babel/traverse": "^7.0.0",
+    "@parcel/babel-ast-utils": "^2.0.0-alpha.3.1",
```

Comment:
It looks like the `import/no-extraneous-dependencies` lint rule didn't catch this. It seems to only be missed with packages in the monorepo?